components/brave_new_tab_ui/brave_new_tab.tsx

```diff
@@ -48,3 +49,10 @@ wireApiEventsToStore()
 
 // Perform DOM-dependent initialization when ready
 document.addEventListener('DOMContentLoaded', initialize)
+
+// Update topsite tiles when NTP gets visible.
+document.addEventListener('visibilitychange', () => {
+  if (document.visibilityState === 'visible') {
+    topSitesAPI.updateMostVisitedInfo()
+  }
+})
```

Comment:
Perhaps for readability we should locate this in the same place as the initial `updateMostVisitedInfo` call? Either straight in `wireApiEventsToStore` or both in a separate `setupHandlerNotifications` function in `topSitesAPI` and that gets called from `apiEventsToStore.ts` or a new `handlerNotifications.ts`?